client/question/components/LearnerViewDirective.js

```diff
@@ -763,6 +820,7 @@ tie.directive('learnerView', [function() {
          * @param {string} questionId ID of question whose data will be loaded
          */
         $scope.loadQuestion = function(questionId) {
+          $scope.isTieThemeSet = true;
```

Comment:
I don't really understand what's going on with this variable -- could you please add some explanation in comments? In particular, isn't the theme always either 'light' or 'dark', and why is isInDarkMode not a sufficient indicator?

Also I suggest $scope.tieThemeIsSet to avoid devs thinking that this is a function (since it starts with a verb).

Reply:
Changed name to pulseAnimationEnabled, which is probably more accurate/descriptive.

Reply:
Added comment. Done.